ios/objc/apis/TorchIValue.h

```diff
@@ -0,0 +1,51 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, TorchIValueType) {
```

Comment:
would be nice to add string/list of strings too - they might occur for text models quite often

in the future, adding dicts would be nice too, but it's lesser priority

Reply:
I can add string type